src/plugins/release-monitor/index.js

```diff
@@ -245,13 +193,7 @@ async function issueCloseHandler(context) {
 
     // check if the closed issue is a release issue
     if (hasReleaseLabel(context.payload.issue.labels)) {
-
-        // remove pending status and add success status on all PRs
-        await createStatusOnAllPRs({
-            context,
-            isSuccess: true,
-            releaseIssueUrl: null
-        });
+        await createStatusOnAllPRs({ context, pendingReleaseIssueUrl: null });
```

Comment:
multi-line objects are more readable. I would recommend changing it to old style.